docs/configuredgraphfactory.txt

```diff
@@ -0,0 +1,459 @@
+[[configuredgraphfactory]]
+ConfiguredGraphFactory
+----------------------
```

Comment:
Please switch away from the old-style Asciidoc headings and use the new-style headings.

Please see [this commit](https://github.com/JanusGraph/janusgraph/pull/115/commits/dc67ce73c08e79fa65b30bc8280861056070c573) for the rationale, along with a Python script that can make the changes for you automatically.